homeassistant/components/nederlandse_spoorwegen/sensor.py

```diff
@@ -35,6 +36,7 @@
         vol.Required(CONF_TO): cv.string,
         vol.Optional(CONF_VIA): cv.string,
         vol.Optional(CONF_TIME): cv.time,
+        vol.Optional(CONF_OFFSET): cv.positive_int,
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration needs to be refactored to use a config flow and config entries.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler